### PR TITLE
Create an endpoint to return the status of API

### PR DESCRIPTION
**Pull request by monoprosito**
Hey Juan!

This change creates a new API to **get the status**.

Also, adds some minor changes:

- .editorconfig file
- .gitignore file

### CODE REVIEW

File: api/v1/views/__init__.py [hunk 1]
```diff
@@ -0,0 +1,3 @@
+from flask import Blueprint
+app_views = Blueprint('app_views', __name__)
```
Comment (johnconnor77) [inline api/v1/views/__init__.py:2]: Url prefix not asigned and  must be at  /api/v1

File: api/v1/app.py [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+@app.teardown_appcontext
+def teardown_db(error):
+    """Closes the database again at the end of the request.
+    """
+    storage.close()
+
+
+if __name__ == "__main__":
+    host = getenv('HBNB_API_HOST', '0.0.0.0')
+    port = getenv('HBNB_API_PORT', 5000)
```
Comment (johnconnor77) [inline api/v1/app.py]: For improving code legibility typing default=value on get env functions "could be better"

**Review by johnconnor77 — CHANGES_REQUESTED** [review]

File: api/v1/views/__init__.py (hunk 1 above)
Comment (monoprosito) [reply api/v1/views/__init__.py:2]: I think that while in the _Blueprint_ instance and in the current part you can send this parameter, for the time being let's test if it works with this current configuration. What do you think?

Comment (johnconnor77) [reply api/v1/views/__init__.py:2]: Ok, Let's move forward

File: api/v1/app.py (hunk 2 above)
Comment (monoprosito) [reply api/v1/app.py]: Of course!

**Comment by johnconnor77** [comment]
Everything is ready for being improve it at master!

**Comment by johnconnor77** [comment]
Everything is ready for being improve it at master!

**Review by johnconnor77 — APPROVED** [review]

**Pull request merged**